moz_minions/minions.py

```diff
@@ -82,6 +82,10 @@ def update(self, **kwargs):
                            }
         info_to_display['name'] = self.name
         self.description = str(info_to_display)
+        self.last_success_cmd = "touch " \
+                                + os.path.join(outdir,
+                                               self.name + "_last_success"
```

Comment:
use self.name as key part of file name will override other concurrent running minions, for example: all minions for mtbf job are naming mtbf_last_success